### ISSUE TITLE
Prevent input being inactive after submit

### ISSUE THREAD

**Issue by manweCZ** [space10-community/conversational-form#187]
Hello, Im trying to figure out how is it possible to keep the user input active and be able to "submit" the form multiple times.

I want to create a chatbot that will first inquire user for some question to "fill the form", and then the Conversational Form would act as a chat (I can append messages from chat already). My only problem is that when I submit the form, the userinput becomes inactive, and when I try to activate it back with

ConversationalForm.userInput.disabled = false;

after resubmitting I get an error

`conversational-form.min.js:4 Uncaught TypeError: Cannot read property 'setTagValueAndIsValid' of undefined
    at e.userInputSubmit (conversational-form.min.js:4)
    at t.dispatchEvent (conversational-form.min.js:1)
    at n.doSubmit (conversational-form.min.js:3)
    at n.onEnterOrSubmitButtonSubmit (conversational-form.min.js:3)
    at n.onKeyUp (conversational-form.min.js:3)
e.userInputSubmit @ conversational-form.min.js:4
t.dispatchEvent @ conversational-form.min.js:1
n.doSubmit @ conversational-form.min.js:3
n.onEnterOrSubmitButtonSubmit @ conversational-form.min.js:3
n.onKeyUp @ conversational-form.min.js:3`

Could you help me? I haven't found anything useful in the examples.

Basically I just need to be able to keep the user input active and working submit, I can handle the rest.

Have a nice day.

**Comment by jenssogaard** [comment]
Hi @manweCZ 

This is not exactly straight forward, but I have mocked an example here where I after first input inject a select for a couple of times before continuing to the last input element.

https://github.com/space10-community/conversational-form/blob/master/docs/dynamic-questions.html

One gotcha, if the user has completed the form it is not possible to use addTags() so keep that in mind. That's why I inject my dynamic questions in between first and last element.

Let me know if you have any questions.
